#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("use_polynomial_fit", bool_t, 0, "Use Polynomial Fit", False)
gen.add("gauss_param_set", bool_t, 0, "set sqr gauss param", False)
gen.add("calc_normal", bool_t, 0, "Calculate Normal", True)

gen.add("polynomial_order", int_t, 0, "Polynomial order", 2, 1, 10)
# bin_size
gen.add("search_radius", double_t, 0, "search radius", 0.03, 0.0, 10.0)

exit (gen.generate (PACKAGE, "jsk_pcl_ros", "MovingLeastSquareSmoothing"))
